x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -313,6 +316,9 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
       getNavLinks$: () => navLinks$,
       setIsSidebarEnabled: (isSidebarEnabled: boolean) =>
         this.isSidebarEnabled$.next(isSidebarEnabled),
+      setGetStartedPage: (getStartedComponent: GetStartedComponent) => {
+        this.getStartedComponent = getStartedComponent;
+      },
```

Comment:
The type is already defined by the plugin, do we need to define the parameter type again? Or can we just:
```suggestion
      setGetStartedPage: (getStartedComponent) => {
        this.getStartedComponent = getStartedComponent;
      },
```